ros2topic/ros2topic/verb/bw.py

```diff
@@ -93,12 +97,19 @@ def __init__(self, node, window_size):
         self.sizes = []
         self.times = []
         self.window_size = window_size
+        self.use_sim_time = node.get_parameter_or(
+            'use_sim_time',
+            rclpy.Parameter('use_sim_time', rclpy.Parameter.Type.BOOL, False)).value
+        if self.use_sim_time:
+            self.clock = node.get_clock()
+        else:
+            self.clock = Clock(clock_type=ClockType.STEADY_TIME)
```

Comment:
This is weird to me. Either it's ok to use a clock that can be adjusted at any time or it is not. If it is ok (required to use sim time ever), then there's no reason that I can think of to use steady time.

Reply:
The default node clock use `ROSTime` which same as `SystemTime` when without `use_sim_time` option, and `SteadyTime` can avoid time jumping in real environment.

Reply:
But why do you need to avoid that? It seems like the logic has to handle jumps either way. I'm not sure sometimes using steady time and sometimes using system time is a good idea.

Reply:
Updated.

Reply:
I was asking the same question before, I think it can go with `SYSTEM_TIME`. Or we can open different PR to discuss this?

Reply:
I have updated the code as https://github.com/ros2/ros2cli/pull/754#discussion_r974832917 metioned.